packages/api/src/types.ts

```diff
@@ -231,6 +231,7 @@ export interface Inputs extends Json {
   env?: string;
   projectId?: string;
   existingResources?: string[];
+  skipNgrok?: boolean;
```

Comment:
Make this extensible? E.g., `depsInfo` or `checkerInfo` or `ignoreCheckers`, etc.